requirements.txt

```diff
@@ -0,0 +1,19 @@
+docutils==0.15.2
+matplotlib==1.5.1
+Jinja2==2.8
+pandas==0.24.2
+six==1.10.0
+sphinx_rtd_theme==0.4.3
```

Comment:
This is only needed for building the docs

Reply:
Removed.